components/data_comps/docs/user-guide/data-ocean-idealized.md

```diff
@@ -0,0 +1,60 @@
+# Data Ocean - Idealized
```

Comment:
I suggest putting the data ocean docs in one file instead of splitting them up.  the TOC mkdocs generates on the rhs will make it easier to navigate the material.

Reply:
I thought it would be too much material for a single page. I was thinking that it would be good to add some plots of my RSO tests to show what it does to the model in a longer run. And for the others I was thinking we might want to eventually add some instructions to generate new SST datasets. Maybe I'm suffering from premature optimization.